frontend/src/App.tsx

```diff
@@ -26,17 +31,30 @@ const App = () => {
 	});
 	// ***PURELY FOR EXAMPLE***
 
+	const { isLoading } = useAuth0()
+
+	if(isLoading){
+		return(
+			<div>
+				<Spinner />
+			</div>
```

Comment:
Use Chakra for UI (i.e. `Box` would be suitable here). Read the Charka docs if you're unfamiliar with their components